src/Workspaces/MSBuild/BuildHost/Rpc/RpcServer.cs

```diff
@@ -25,19 +26,19 @@ namespace Microsoft.CodeAnalysis.MSBuild;
 /// </remarks>
 internal sealed class RpcServer
 {
-    private readonly TextWriter _sendingStream;
+    private readonly TextWriter _streamWriter;
     private readonly SemaphoreSlim _sendingStreamSemaphore = new SemaphoreSlim(initialCount: 1);
-    private readonly TextReader _receivingStream;
+    private readonly TextReader _streamReader;
 
     private readonly ConcurrentDictionary<int, object> _rpcTargets = [];
     private volatile int _nextRpcTargetIndex = -1; // We'll start at -1 so the first value becomes zero
 
     private readonly CancellationTokenSource _shutdownTokenSource = new CancellationTokenSource();
 
-    public RpcServer(Stream sendingStream, Stream receivingStream)
+    public RpcServer(PipeStream stream)
```

Comment:
This change wasn't necessary, but keeps it consistent with RpcClient only accepting a PipeStream.